### DISCUSSION TITLE
PhpMyAdmin

### DISCUSSION THREAD

**Discussion by farshadl** [Gozargah/Marzban#883] [Q&A]
سلام. من چطور میتونم وارد پنل PhpMyAdmin بشم؟
تو مرزبان‌نامه نوشته بودن که از پورت ۸۰۱۰ باید استفاده کرد. من ساب دامنه پنلم را بصورت xxx.example.com:8010 میزنم ولی هیچی بالا نمیاد.
میخوام از query استفاده کنم ولی نمیتونم وارد پنل PhpMyAdmin بشم.

**Comment by mdjvd** [comment]
سلام. اگر با دامنه میخواید وارد بشید نباید تیک پروکسی دامنه روشن باشه و آدرس رو باید به صورت http بزنید.
میتونید به جای دامنه، آیپی خود سرور رو بذارید، مثلا:
1.2.3.4:8010

**Comment by TIR3D4** [comment]
سلام وقت شما بخیر متاسفانه منم مشکل باز نشدن phpmyadmin و دارم تمام مراحل رو درست انجام دادم قبلا برام باز میشد اما مدتی هست کلا باز نمیشه 
با دامین http میزنم با پورت 8010 و پروکسی کلودفلر خاموش اما باز نمیشه با ایپی هم باز نمیشه هرکاری هم کردم درست نمیشه لطفا راهنمایی کنید

**Reply by mdjvd** [reply]
شاید ایپی سرور فیلتر شده، با vpn امتحان کنید
از باز بودن پورت 8010 توی فایروال سرور (اگر فعاله) هم مطمئن بشید
با marzban status هم میتونید وضعیتش رو چک کنید
